packages/graph/hash_graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -0,0 +1,328 @@
+use std::{fmt, fmt::Write};
+
+use postgres_types::ToSql;
+
+use crate::store::{
+    postgres::query::{
+        database::{Column, Table, TableAlias},
+        Path, PostgresQueryRecord, Transpile,
+    },
+    query::{Filter, FilterValue, Parameter},
+};
+
+/// A [`Filter`] compiled to postgres.
```

Comment:
The struct itself doesn't represent a compiled `Filter`, right? We would have to call `transpile` to get the SQL out, so is it misleading to have this docstring?